APIKit/APIKit.swift

```diff
@@ -43,6 +39,41 @@ public class API {
     public class func responseBodyParser() -> ResponseBodyParser {
         return .JSON(readingOptions: nil)
     }
+    
+    public class func URLSessionConfiguration() -> NSURLSessionConfiguration {
+        return NSURLSessionConfiguration.defaultSessionConfiguration()
+    }
+    
+    // prevent instantiation
+    override private init() {
+        super.init()
+    }
+    
+    // create session and instance of API for each subclasses
+    private final class var instancePair: (API, NSURLSession) {
+        struct Singleton {
+            static var dictionary = [String: (API, NSURLSession)]()
+        }
```

Comment:
How about using private, global scope variable until we can use stored class var in Swift 1.2?

```swift
private var instancePairDictionary = [String: (API, NSURLSession)]()

public class API {
}
```